x-pack/metricbeat/module/sql/query/query.go

```diff
@@ -44,24 +51,32 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 	cfgwarn.Beta("The sql query metricset is beta.")
 
 	config := struct {
-		Driver string `config:"driver" validate:"nonzero,required"`
-		Query  string `config:"sql_query" validate:"nonzero,required"`
+		Driver            string `config:"driver" validate:"nonzero,required"`
+		Query             string `config:"sql_query" validate:"nonzero,required"`
+		SQLResponseFormat string `config:"sql_response_format" validate:"nonzero,required"`
```

Comment:
Yeah, that was something I was thinking about, even I remember that I did it that way but after parsing the `sql.yml` the value becomes empty, but let me give it a try because I don't remember well. But the thing is to allow a default value (`table`) for `response_format` option.